Sources/Purchasing/NonSubscriptionTransaction.swift

```diff
@@ -34,11 +34,12 @@ public final class NonSubscriptionTransaction: NSObject {
     /// The unique identifier for the transaction created by the Store.
     @objc public let storeTransactionIdentifier: String
 
-    /**
-     * The ``Store`` where this transaction was performed.
-     */
+    /// The ``Store`` where this transaction was performed.
     @objc public let store: Store
 
+    /// The display name of the product as configured in the RevenueCat dashboard.
+    @objc public let displayName: String?
```

Comment:
I remember we had a conversation about this, and decided that this might not be something the developer wants to show to their customers 🤔 
Right?

Reply:
yeah, but the alternative is the product id. We would only get here for other store products. I think it's a good compromise.

I would say let's leave this and if someone complains we can change it

Reply:
Oh this probably answer my previous comment. However, this property would not be localized, so could it mean trouble if developers rely on this property to be user-facing and contains a localization the don't expect? Just asking